src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
 public class Application {
+    private static int COUNT_OF_BALL = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] lotteries = new Lottery[COUNT_OF_BALL];
```

Comment:
We do not need many lotteries. Same as we do not need 3 shops to buy 3 milk bottles.